neurobooth_os/tasks/task.py

```diff
@@ -361,6 +361,31 @@ def doDriftCorrect(self, vals):
         if self.eye_tracker is not None:
             self.eye_tracker.tk.doDriftCorrect(*vals)
 
+    def update_tablet_background(self, x: float, y: float) -> None:
+        '''
+           Draw a cross and a box on the eyelink tablet.
+           x and y are positions in 0 centered psychopy coordinate space.
+           x and y can be int or float
+           See pos_psych2pix above
+        '''
+
+        # First clear tablet screen
+        self.sendCommand('clear_screen 0')
+
+        # convert from 0 centered to top-left centered coordinate space
+        top_left_xy = self.pos_psych2pix([x, y])
+
+        # draw cross at top_left centered x,y position
+        self.sendCommand('draw_cross %d %d 10' % tuple(top_left_xy))
```

Comment:
I would like to factor this out into its own quick method `tablet_draw_cross(x, y, whatever_this_is=10)` in case we ever want to use it again for some other reason.